code-security/admin_guide/get-started/connect-your-repositories/add-intellij.adoc

```diff
@@ -0,0 +1,53 @@
+:topic_type: task
+
+[.task]
+== Add IntelliJ to Prisma Cloud Code Security
+
+Integrating IntelliJ with Prisma Cloud Code Security makes it possible for you to identify misconfigurations before you commit your code, and avoid pull requests that can potentially fail builds due to undetected misconfigurations. Using Checkov, a code analysis tool that scans Infrastructure-as-code (IaC) files from frameworks such as Terraform, CloudFormation, and Azure Resource Manager (ARM) on IntelliJ gives you immediate detection of misconfigurations and inline code fixes.
+Apart from the scanning the common industry standard policies, Checkov also identifies custom policies that you can configure during <<_add-a-new-custom-policy-for-build-time-checks,custom build-time checks>>.
+
+Before you begin with IntelliJ integration, you must first <<_generate-access-keys,generate access keys>> and have https://www.python.org/downloads/[Python] version 3.7 or above installed, as Checkov runs on Python. Alternatively, https://docs.pipenv.org/[Pipenv] or https://www.docker.com/products/docker-desktop[Docker] is also recommended. A configuration of Checkov plugin on IntelliJ during the code build with IaC security feedback within code reviews and CI/CD pipelines, helps you catch misconfigurations before your code is merged centrally.
```

Comment:
Are we saying to use the Checkov plugin on IntelliJ and the IntelliJ integration for Code Security?

Reply:
Yes. Since it is Checkov CLI that actually scans the IaC files. And then when the IDEs are integrated with Prisma Cloud, the custom policies can also be scanned. The integration between IDE and PC is mainly to access the custom policies.